winfsp_wrs/src/callback.rs

```diff
@@ -12,11 +14,75 @@ use crate::{
         SECURITY_INFORMATION, SIZE_T, UINT32, UINT64, ULONG,
     },
     CleanupFlags, CreateFileInfo, CreateOptions, DirInfo, FileAccessRights, FileAttributes,
-    FileInfo, PSecurityDescriptor, SecurityDescriptor, VolumeInfo,
+    FileContextMode, FileInfo, PSecurityDescriptor, SecurityDescriptor, VolumeInfo,
 };
 
+/// Implement only if necessary at your own risk
+pub trait FileContextKind {
+    fn mode() -> FileContextMode;
```

Comment:
Can we use an [associated constant](https://doc.rust-lang.org/reference/items/associated-items.html#associated-constants) instead ?

```rust
trait FileContextKind {
  const MODE: FileContextMode;
...
}
```

Reply:
Yes, nice suggestion